packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -58,9 +58,15 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
 
   /**
    * Element type the root element will use. Default is "div".
+   * @deprecated Use 'as' instead.
    */
   elementType?: keyof React.ReactHTML;
 
+  /**
+   * A component that should be used as the root element of the link returned from the Link component.
```

Comment:
Why does this comment reference the `Link` component?

Reply:
Thank you for finding this! Just pushed the update with it properly reflecting that's an element of the FocusZone component.